iina/PlayerCore.swift

```diff
@@ -2158,7 +2158,7 @@ class PlayerCore: NSObject {
 
   func sendOSD(_ osd: OSDMessage, autoHide: Bool = true, forcedTimeout: Float? = nil, accessoryView: NSView? = nil, context: Any? = nil, external: Bool = false) {
     // querying `mainWindow.isWindowLoaded` will initialize mainWindow unexpectly
-    guard mainWindow.loaded && Preference.bool(for: .enableOSD) else { return }
+    guard mainWindow.loaded, Preference.bool(for: .enableOSD) || osd.alwaysEnabled else { return }
```

Comment:
Why using `,` here instead of `&&`?

Reply:
I tried to find what Swift considers the best practice in this case and only found people debating that. The advice in the Medium post [Chaining logical expressions with commas in Swift](https://medium.com/@austinlouden/chaining-logical-expressions-with-commas-in-swift-c3755ee784e9) about  expressions using commas being easier to read than complex expressions reflects what I was thinking. Using `&&` would require adding `()` around the `||` expression. I was also thinking that the comma was more like swift has you have to use it when using `let`.

I'm good with whatever style the group wants to use. Let me know if you want me to switch this to `&&`.

Reply:
I was just asking and also found it a little bit confusing. I think it's alright to use `,` here to avoid the parentheses.